_locales/en/messages.json

```
@@ -162,14 +162,29 @@
     "message": "Scratch Addons theme"
   },
   "scratchAddonsThemeDescription": {
-    "message": "This allows you to switch between light and dark themes for this settings page, and the extension's popup."
+    "message": "Choose the style of the Scratch Addons page to light or dark. You can set the mode to automatic options as change at specific time, or to be the same as in your system settings. Automatic settings can also turn on and off add-ons like \"Editor Dark Mode\" and \"Dark WWW\"."
   },
   "light": {
     "message": "Light"
   },
   "dark": {
     "message": "Dark"
   },
+  "autoTheme": {
+    "message": "Sync with system"
+  },
+  "timeTheme": {
+    "message": "Toggle at sunset/sunrise"
+  },
+  "sunset": {
+    "message": "Sunset"
+  },
+  "sunrise": {
+    "message": "Sunrise"
+  },
+  "syncAddonsWithTheme": {
+    "message": "Make dark mode addon sync with these settings"
```

I have another idea for the name of this:
```suggestion
    "message": "Disable dark mode addons when using light theme"
```